mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+import {useState} from "react";
+import clsx from "clsx";
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ * {@link Checkbox}
+ */
+interface CheckboxProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Checkbox`s value
+   */
+  value?: string;
+  /**
+   * Checkbox's name
+   */
+  name?: string;
+  /**
+   * Checkbox's class
+   */
+  className?: string | string[];
```

Comment:
so how to use this component? how to handle changes? don't see any appropriate property

Reply:
onChange propery was before